lib/IR/Builders.cpp

```diff
@@ -297,6 +297,10 @@ IntegerSet Builder::getIntegerSet(unsigned dimCount, unsigned symbolCount,
   return IntegerSet::get(dimCount, symbolCount, constraints, isEq);
 }
 
+AffineMap Builder::getEmptyAffineMap(MLIRContext *context) {
+  return AffineMap::get(context);
```

Comment:
Builder stores a context, there is no need to pass another context in this call.